packages/botonic-react/src/webchat-app.tsx

```diff
@@ -248,21 +187,35 @@ export class WebchatApp {
     this.onClose && this.onClose(this, ...args)
   }
 
-  async onUserInput({ user, input }) {
+  async onUserInput({ user, input }: OnUserInputArgs): Promise<void> {
+    if (!user) return
```

Comment:
if `Botonic.addUserText()` is not called with user and the function goes through onUserInput, maybe when calling it it won't be executed. Can you review it?

Reply:
Botonic.addUserText() -> Botonic.addUserMessage() -> webchatRef.current?.addUserMessage(message) -> sendInput() -> sendUserInput() -> onUserInput()